packages/oas-to-har/src/index.js

```diff
@@ -14,11 +15,17 @@ function formatter(values, param, type, onlyIfExists) {
     return undefined;
   }
 
-  if (param.required && param.example) {
-    return param.example;
+  if (param.required && param.schema && param.schema.default) {
+    return param.schema.default;
   }
 
-  return param.name;
+  // If we don't have any values for the path parameter, just use the name of the parameter as the value so we don't
+  // try try to build a URL to something like `https://example.com/undefined`.
+  if (type === 'path') {
+    return param.name;
+  }
+
+  return undefined;
```

Comment:
Made this change because for our own API definition, if a user didn't specify an `x-readme-version` header, we'd send a header that evaluated to `x-readme-version: 'x-readme-version'` which isn't right at all.